OrbitCore/CrashHandler.h

```diff
@@ -12,13 +14,14 @@ class CrashHandler {
  public:
   explicit CrashHandler(const std::string& dump_path,
                         const std::string& handler_path,
-                        const std::string& crash_server_url,
-                        bool is_upload_enabled);
+                        const std::string& crash_server_url);
   void DumpWithoutCrash() const;
+  void SetUploadsEnabled(bool is_upload_enabled) const;
```

Comment:
Don't make this const even though it's technically correct. The better solution would be to not put crash_report_db_ in a unique_ptr but sadly that's not possible due to bad API.